src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/command/flow/InstallIngressFlow.java

```diff
@@ -143,6 +154,7 @@ public InstallIngressFlow(@JsonProperty(TRANSACTION_ID) final UUID transactionId
         setMeterId(meterId);
         setEnableLldp(enableLldp);
         setEgressSwitchId(egressSwitchId);
+        this.enableArp = enableArp;
```

Comment:
nit: align it with lldp(there is a setter for the lldp above)

Reply:
fixed